build.gradle.kts

```diff
@@ -1,38 +1,20 @@
+import io.gitlab.arturbosch.detekt.Detekt
+import org.jetbrains.kotlin.gradle.dsl.JvmTarget.JVM_21
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+
 plugins {
-  id("uk.gov.justice.hmpps.gradle-spring-boot") version "6.0.9"
+  id("uk.gov.justice.hmpps.gradle-spring-boot") version "6.1.2"
   kotlin("plugin.spring") version "2.0.21"
   kotlin("plugin.jpa") version "2.0.21"
   id("io.gitlab.arturbosch.detekt") version "1.23.7"
 }
 
-configurations {
-  testImplementation { exclude(group = "org.junit.vintage") }
-}
-
-val integrationTest = task<Test>("integrationTest") {
-  description = "Integration tests"
-  group = "verification"
-  shouldRunAfter("test")
-}
-
-tasks.register<Copy>("installLocalGitHook") {
-  from(File(rootProject.rootDir, ".scripts/pre-commit"))
-  into(File(rootProject.rootDir, ".git/hooks"))
-  filePermissions { unix(755) }
-}
-
-tasks.named<Test>("integrationTest") {
-  useJUnitPlatform()
-  filter {
-    includeTestsMatching("*.integration.*")
-  }
+repositories {
+  mavenCentral()
 }
 
-tasks.named<Test>("test") {
-  filter {
-    excludeTestsMatching("*.integration.*")
-  }
-}
+ext["hibernate.version"] = "6.5.3.Final"
```

Comment:
need to pin to this version of hibernate as there's a bug around not being able to modify ids 
similar to AFER